instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/auto/instrumentation/awssdk/v1_11/AwsSdkClientTracer.java

```diff
@@ -64,20 +65,27 @@ public Span onRequest(final Span span, final Request<?> request) {
     span.setAttribute("aws.operation", awsOperation.getSimpleName());
     span.setAttribute("aws.endpoint", request.getEndpoint().toString());
 
-    if (contextStore != null) {
-      RequestMeta requestMeta = contextStore.get(originalRequest);
-      if (requestMeta != null) {
-        span.setAttribute("aws.bucket.name", requestMeta.getBucketName());
-        span.setAttribute("aws.queue.url", requestMeta.getQueueUrl());
-        span.setAttribute("aws.queue.name", requestMeta.getQueueName());
-        span.setAttribute("aws.stream.name", requestMeta.getStreamName());
-        span.setAttribute("aws.table.name", requestMeta.getTableName());
-      }
+    if (requestMeta != null) {
+      span.setAttribute("aws.bucket.name", requestMeta.getBucketName());
+      span.setAttribute("aws.queue.url", requestMeta.getQueueUrl());
+      span.setAttribute("aws.queue.name", requestMeta.getQueueName());
+      span.setAttribute("aws.stream.name", requestMeta.getStreamName());
+      span.setAttribute("aws.table.name", requestMeta.getTableName());
     }
-
     return span;
   }
 
+  /**
+   * Override startScope not to inject context into the request since no need to propagate context
```

Comment:
Still don't understand why do we do something extra to avoid it.

Reply:
super class's startScope will do 2 things.. get the span and then inject the context into the request.  this is not needed for aws sdk tracer.

Reply:
That's precisely what I don't understand: why do we want to avoid injecting trace context headers into AWS SDK requests? Even if in most cases nobody will consume them, what bad will happen if we do inject them? Is it worth making our code more complicated?

Reply:
that's not my decision.. it was like that before.  please talk to the creator of aws sdk decorator.  it's out of scope for this pr anyway.